### ISSUE TITLE
[MARCHETYPES-63] upgrade plugins versions to avoid failure with Java 11

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-archetypes#101]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** opened **[MARCHETYPES-63](https://issues.apache.org/jira/browse/MARCHETYPES-63?redirect=false)** and commented

projects created from current archetypes should build with Java 11, which is the current LTS

tests show that at least maven-surefire-plugin has to be upgraded (currently using 2.20.1)

---

**Affects:** 1.3

**Issue Links:**
- [MNGSITE-360](https://issues.apache.org/jira/browse/MNGSITE-360) upgrade archetype version to 1.4 to support Java 11
 (_**"is depended upon by"**_)

**Comment by jira-importer** [comment]
**[Bernd Eckenfels](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=b.eckenfels)** commented

Surefire 2.22.1 worked for me.
 
BTW: the site build also prints a warning about missing version and fails with Java 11, should we describe that also or just fix it in 1.4?

Add:

```xml
 <plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-project-info-reports-plugin</artifactId>
    <version>2.7</version>
  </plugin>
```

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

surefire upgrade in the archetype (with compiler and javadoc plugins) already done in https://gitbox.apache.org/repos/asf?p=maven-archetypes.git&a=commit&h=6121eac6da6d556ca3ec7284824f5d26d2ae74e6 (without Jira issue)
